CMakeLists.txt

```diff
@@ -1229,6 +1230,17 @@ if(WITH_TESTS)
 endif()
 
 if(WITH_BENCHMARK_TOOLS)
+  if(NOT WITH_TESTS)
+    add_subdirectory(third-party/gtest-1.8.1/fused-src/gtest)
+    add_library(testharness STATIC
+    test_util/testharness.cc)
+    target_link_libraries(testharness gtest)
+    set(TESTS
```

Comment:
I think `TESTS` is probably not needed here.

Reply:
removed @ajkr